django/core/checks/urls.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from . import Tags, Warning, register
+
+W001 = Warning(
+    "In your url patterns, you have used include with a regex containing a $. "
```

Comment:
I'm not too keen on beginning each warning with "In your url patterns, ..".

How about "Your url patterns .." ?

"Your url patterns have used `include` with a regex containing a '$'. " .. 

"Your url patterns have a regex beginning with a '/'." ..

"Your url patterns have a pattern with a name containing a ':'." ..